htslib/kstring.h

```diff
@@ -68,11 +68,11 @@
 /* kstring_t is a simple non-opaque type whose fields are likely to be
  * used directly by user code (but see also ks_str() and ks_len() below).
  * A kstring_t object is initialised by either of
- *       kstring_t str = { 0, 0, NULL };
- *       kstring_t str; ...; str.l = str.m = 0; str.s = NULL;
+ *       kstring_t str = KS_INITIALIZE;
+ *       kstring_t str; ...; ks_initialize(&str);
  * and either ownership of the underlying buffer should be given away before
  * the object disappears (see ks_release() below) or the kstring_t should be
- * destroyed with  free(str.s);  */
+ * destroyed with  ks_free(&str);  */
```

Comment:
Might be fair to bow to legacy reality with

    destroyed with  ks_free(&str)  or  free(str.s)